src/app/dashboard/userStockList.html

```diff
@@ -0,0 +1,17 @@
+<div class="paperCard" ng-repeat="stock in stocks.stocks">
+	<h1>{{ stock.title }}</h1>
+	<div>
+		<p class="value">{{ stock.value | currency }}</p>
+		<p class="change">Change: {{ stock.change | currency }}</p>
+		<div class="models">
+			<span class="good"></span>
```

Comment:
are these the 5 parts or economic indicators?
we should mock this data from your StockService and use ng-class to determine which class to display and combine that with an ng-repeat for the 5 indicators